api/src/main/java/io/onetable/spi/sync/TargetClient.java

```diff
@@ -85,4 +88,10 @@ public interface TargetClient {
 
   /** Returns the {@link io.onetable.model.storage.TableFormat} the client syncs to */
   TableFormat getTableFormat();
```

Comment:
Would it be easier or clearer to just make this return a string instead of adding the `getTableFormatName` below? I'm fine with either method name but I think it makes sense to only have one or the other.

Reply:
I was concerned about the existing enduser code, tests, blogs, demos, etc - so I was trying to keep both.

Reply:
The first release is marked as a beta with a note that APIs may change. This early on I don't think we have anyone using this in production besides my team.